cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -287,7 +287,7 @@ module Cardano.Api.Typed (
 
     -- ** Script addresses
     -- | Making addresses from scripts.
-    ScriptHash,
+    ScriptHash(..),
```

Comment:
We should not need access to this constructor. The API can already provide what we need, see below.